### PR TITLE
Rewrite Concatenation Guide for Clarity

### PR DESCRIPTION
**Pull request by marklewin**
## Description

Light rewrite for clarity.

## Deploy Notes

Test in sample app. No dependencies on building blocks, etc

### CODE REVIEW

File: _documentation/messaging/sms/guides/concatenation-and-encoding.md [hunk 1]
```diff
@@ -29,31 +29,33 @@ a b c d e f g h i j k l m n o p q r s t u v w x y z
 Ä Å Æ Ç É Ñ Ø ø Ü ß Ö à ä å æ è é ì ñ ò ö ù ü Δ Φ Γ Λ Ω Π Ψ Σ Θ Ξ
 ````
 
-Characters from the [extended table](https://en.wikipedia.org/wiki/GSM_03.38#GSM_7-bit_default_alphabet_and_extension_table_of_3GPP_TS_23.038_/_GSM_03.38) require two bytes per character to encode, the first prefix character is ESC followed by the character from the extended table:
+Characters from the [extended table](https://en.wikipedia.org/wiki/GSM_03.38#GSM_7-bit_default_alphabet_and_extension_table_of_3GPP_TS_23.038_/_GSM_03.38) require two bytes per character to encode, the first prefix character is `ESC` followed by the character from the extended table:
```
Comment (mheap) [inline _documentation/messaging/sms/guides/concatenation-and-encoding.md]: What does this mean for the customer? Does each of these characters count as two towards our limit of 160 characters?

File: _documentation/messaging/sms/guides/concatenation-and-encoding.md [hunk 2]
```diff
@@ -4,20 +4,20 @@ title: Concatenation and Encoding
...
+If you send a message that contains more than the maximum number of characters per message for your chosen encoding, Nexmo sends a *concatenated SMS*. A concatenated SMS contains multiple SMS parts that are connected by segmentation information in the [User Data Header (UDH)](https://en.wikipedia.org/wiki/User_Data_Header).
 
-Segmentation information tells the handset the number of messages that make up the concatenated SMS, and the position of each SMS part in the concatenated SMS. When the handset has received all the SMS parts, it presents your message as a single text to your user.
+This segmentation information tells the handset the number of message parts that make up the concatenated SMS, and the position of each SMS part within it. When the handset has received all of the SMS parts, it presents the message to the recipient as a single text.
 
 ## Encoding
 
-There are two main types of messages supported by the Nexmo SMS API:
+Specify the encoding by setting the `type` parameter in the request. The encoding you should use depends on the characters that the message contains:
```
Comment (mheap) [inline _documentation/messaging/sms/guides/concatenation-and-encoding.md]: It feels like we're jumping in to how to set an encoding when I don't understand what it is. The numbered items below help explain it, but the previous sentence of `There are two main types of messages supported by the Nexmo SMS API` helped set the scene

File: _documentation/messaging/sms/guides/concatenation-and-encoding.md [hunk 3]
```diff
@@ -29,31 +29,33 @@ a b c d e f g h i j k l m n o p q r s t u v w x y z
...
 
-Languages such as Arabic, Chinese, Korean, Japanese, or Cyrillic alphabet languages require Unicode characters beyond the GSM standard and extended ranges. These characters require 16-bit UCS-2 encoding. Messages sent using Unicode characters must have their `type` set to `unicode`. Note that in this case all characters in the message will be encoded using UCS-2, regardless of their location in the Unicode table.
+Languages such as Arabic, Chinese, Korean, Japanese, or Cyrillic alphabet languages require Unicode characters beyond the GSM standard and extended ranges. These characters require 16-bit UCS-2 encoding. 
+
+If your message includes any of these characters, you must set the `type` to `unicode`. When you choose this encoding, *all* characters within the message are encoded using UCS-2, even if they are present in the GSM standard or extended character sets.
 
 ## Examples
 
-| Message | Type | Bytes used per character | Characters used in message |
+| Message | Type | Bytes per character | Character set used in messagee |
```
Comment (mheap) [inline _documentation/messaging/sms/guides/concatenation-and-encoding.md]: s/messagee/message

File: _documentation/messaging/sms/guides/concatenation-and-encoding.md [hunk 4]
```diff
@@ -29,31 +29,33 @@ a b c d e f g h i j k l m n o p q r s t u v w x y z
...
 |---|---|---|---|
 | `Bonjour monde` | `text` | 1 | GSM standard |
 | `This ^ That ` | `text` | 1 and 2 for `^` | GSM standard and GSM extended |
 | `こんにちは世界` | `unicode` | 2 (UCS-2) | Unicode | 
 
 ## Maximum number of characters
 
-The maximum number of characters you can fit into an SMS part depends on the [Encoding](#encoding) that you are using.
+The maximum number of characters you can fit into a single SMS depends on the type of encoding.
```
Comment (mheap) [inline _documentation/messaging/sms/guides/concatenation-and-encoding.md]: I think SMS part was technically correct here, as an SMS can be one or more parts, and the character limitation (80, 160 etc) is per-part

File: _documentation/messaging/sms/guides/concatenation-and-encoding.md [hunk 5]
```diff
@@ -62,20 +64,22 @@ If you are sending a message with type value of `text` then the following charac
...
 
-> Note: You are charged for each SMS sent as part of a concatenated SMS.
+> Note: You are charged for each SMS part in a concatenated SMS.
 
 ## Inbound Messages
 
-If you rent one or more virtual numbers from Nexmo, inbound SMS to that number are sent to your webhook endpoint using either GET or POST. Inbound messages comply with the SMS specification. If an SMS sent to your virtual number is longer than the [maximum number of characters](#maximum-number-of-characters) for an individual SMS, you should receive the concatenated message in SMS parts.
+Inbound messages comply with the SMS specification. If an SMS sent to your virtual number is longer than the [maximum number of characters](#maximum-number-of-characters) allowed for a single SMS, you will receive the concatenated message in parts.
+
+However, you can only receive inbound concatenated SMS if the carrier your user is sending through supports them. If the carrier does not support concatenation then a useful workaround is to treat all SMS from the same number within a short time window as concatenated.
```
Comment (mheap) [inline _documentation/messaging/sms/guides/concatenation-and-encoding.md]: What do we class as a short time window? Is this a recommendation from support?

File: _documentation/messaging/sms/guides/concatenation-and-encoding.md (hunk 1 above)
Comment (marklewin) [reply _documentation/messaging/sms/guides/concatenation-and-encoding.md]: Yes it does. I have now (hopefully) made this very clear.

**Comment by mheap** [comment]
@marklewin I've pushed a few changes to clarify how the extended character set works. If you're happy with them, please squash and merge

**Pull request merged**